embedded-visualization/Actions/cspe-Pending-Resize-Actions-Summary.json

```diff
@@ -1152,12 +1152,11 @@
         "allValue": null,
         "current": {
           "selected": true,
-          "tags": [],
           "text": [
-            "All"
+            "HawthorneSales/SEDEMO"
```

Comment:
why should we harcdcode our demo cluster in a report?

Reply:
This is how Grafana exports a report. It keeps the value which are live when you do the export. However those fields are based on a variables which takes the response of a live query which will change depending on the system it's imported on.

Reply:
but if you want to publish it for customers, would you want to keep the "All" in your code?

Reply:
Yes, because the "All" is a meta value which provides the ability to select all the values at once. You can have "Group A", "Group B", "Group C" and "All" when you want to display values for A, B and C at once.